pkg/client/cmd/deploy.go

```diff
@@ -297,7 +303,7 @@ func sendAppTarball(appName, appFolder string, stream dpb.Deploy_MakeClient) err
 			Chunk: buf,
 		}}}
 		if err := stream.Send(bufMsg); err != nil {
-			fmt.Fprintln(os.Stderr, "Error sending tarball chunk:")
+			fmt.Fprintln(os.Stderr, "Error sending tarball chunk: %v", err)
```

Comment:
maybe`Fprintf` or `Fprintln(os.Stderr, "Error sending tarball chunk:", err)` ?

Reply:
yeah I'm going blind for sure...

Reply:
😂